docs/tutorials/static-types.md

```diff
@@ -123,6 +123,14 @@ params {
 
 See {ref}`workflow-params-def` for more information about the `params` block.
 
+:::{note}
+Parameters that are only used in the config file should be declared in the config file, not the script. Since rnaseq-nf does not have any such parameters, all parameters are declared in the script. See {ref}`config-params` for more information.
```

Comment:
```suggestion
Parameters used only in the config file should be declared there, not in the script. Since rnaseq-nf has no such parameters, all parameters are declared in the script. See {ref}`config-params` for more information.
```